content/guides/core/artifacts/create-a-new-artifact-version.md

```diff
@@ -38,12 +38,12 @@ Based on your use case, select one of the tabs below to create a new artifact ve
   {{% tab header="Inside a run" %}}
 Create an artifact version within a W&B run:
 
-1. Create a run with `wandb.init`. (Line 1)
-2. Create a new artifact or retrieve an existing one with `wandb.Artifact` . (Line 2)
-3. Add files to the artifact with `.add_file`. (Line 9)
-4. Log the artifact to the run with `.log_artifact`. (Line 10)
+1. Create a run with `wandb.init`.
+2. Create a new artifact or retrieve an existing one with `wandb.Artifact`.
+3. Add files to the artifact with `.add_file`.
+4. Log the artifact to the run with `.log_artifact`.
 
-```python showLineNumbers
+```python 
```

Comment:
To put them back in the Hugo way, you could instead add this right after the backticks to start the code fence: ```{linenos=true}``` If you also want to highlight some lines, you can add something like ```{hl_lines=[5]}```.

Reply:
Right. I am aware of that. :) I removed them because of the weird CSS happening https://wandb.atlassian.net/browse/DOCS-1354. [Different kind of highlighting]